common/styles/utilities/_root-scope-classes.scss

```diff
@@ -423,6 +423,22 @@
   width: auto;
 }
 
+.image-ls-overlay {
```

Comment:
what does `ls` mean?

Reply:
…and where are these classes being used?

Reply:
It was `L`-s, as in the plural of `L`.
It's not and it's been removed.